sherpa/astro/ui/utils.py

```diff
@@ -6564,6 +6564,18 @@ def set_analysis(self, id, quantity=None, type='rate', factor=0):
         they are interpreted as the `id` and `quantity` parameters,
         respectively.
 
+        Sherpa does not match XSPEC when the `quantity` setting is set
+        to "channel", particularly when the data is grouped. Sherpa
```

Comment:
I would remove "Sherpa does no match XSPEC..." sentence and reference to XSPEC later on. This is Sherpa documentation and some people may not even know what XSPEC is (really :-))

Reply:
So, is this whole paragraph not worth including then? I think it might be useful to have something that explains why our plots and XSPEC look different.  I put it at the end of the notes section so it's not the first thing a user sees.

For example here's XSPEC's equivalent of `plot_data`

![xspec](https://github.com/sherpa/sherpa/assets/224638/1a4a8b02-46ab-42ba-b699-c5c2a3c30270)

and here's our version:

![sherpa](https://github.com/sherpa/sherpa/assets/224638/2605005d-3645-4bd2-b73c-a13c265470d0)

Reply:
This is the **same** dataset (our favourite `sherpa-test-data/sherpatest/3c273.pi` for anyone who wants to play along at home).

Reply:
Perhaps a better place would be to include this in the CIAO threads or FAQ on CIAO pages.